pkgs/applications/misc/syncthing_exporter/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, fetchFromGitHub, buildGoModule }:
+
+buildGoModule rec {
+  pname = "syncthing_exporter";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "f100024";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-C3YnGs7A19No7tY7tIilwKi55n4S8x5whfyiAqMxMj0=";
+  };
+
+  vendorSha256 = "sha256-JbrCgt5Tu65IZNqtz3OvIESwYO1p6cH1ohzkVc1pwTE=";
+
+  postInstallCheck = ''
+    $out/bin/syncthing_exporter --version > /dev/null
+  '';
+
+  meta = with lib; {
+    description = "Syncthing metrics exporter for prometheus";
```

[Prometheus](https://prometheus.io/) is using `Prometheus` not `prometheus`. We should respect their way of using the name.

```suggestion
    description = "Syncthing metrics exporter for Prometheus";
```